src/System.Private.DataContractSerialization/src/System.Private.DataContractSerialization.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyName>System.Private.DataContractSerialization</AssemblyName>
-    <AssemblyVersion>4.1.1.0</AssemblyVersion>
+    <AssemblyVersion>4.1.1.1</AssemblyVersion>
```

Comment:
Just a NIT, I would prefer to give the assembly the same version of the package (ie. 4.1.2.0) for easy remember. It may make our life a little bit easier in the future in debugging given that there are so many different versions floating around.